ui/src/timeMachine/components/FunctionSelector.tsx

```diff
@@ -15,22 +15,44 @@ import {
 } from 'src/timeMachine/actions/queryBuilder'
 
 // Utils
-import {getActiveQuery} from 'src/timeMachine/selectors'
+import {getActiveQuery, getIsInCheckOverlay} from 'src/timeMachine/selectors'
 
 // Constants
 import {
   FUNCTIONS,
   AGG_WINDOW_AUTO,
+  AGG_WINDOW_NONE,
 } from 'src/timeMachine/constants/queryBuilder'
 
 // Types
 import {AppState, BuilderConfig} from 'src/types'
 
 const FUNCTION_NAMES = FUNCTIONS.map(f => f.name)
 
+const DURATIONS = [
```

Comment:
Maybe `DURATIONS` should be equal to the `CHECK_EVERY_OPTIONS` const?

Reply:
That seems better, I've updated it.